dist/rust/riot-rust-linker-hook

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Iff (sic, please argue for it) bash is really needed, at least invoke it like `#!/usr/bin/env bash`.